### ISSUE TITLE
No newline between multiple cf outputs

### ISSUE THREAD

**Issue by srbry** [EngineerBetter/cf-plex#2]
Running cf-plex orgs can return things like:

```
Running 'cf orgs on api'
Getting orgs as user...

name
system
org1
org2
Running 'cf orgs on api'
Getting orgs as user...

name
system
org1
org2
```

It would be easier to read with a new line after the previous cfs output, for example:

```
Running 'cf orgs on api'
Getting orgs as user...

name
system
org1
org2

Running 'cf orgs on api'
Getting orgs as user...

name
system
org1
org2
```

**Comment by DanielJonesEB** [comment]
On it: https://www.pivotaltracker.com/story/show/121029515

**Comment by DanielJonesEB** [comment]
Fixed in v1.2.0
